slam-ws/src/nav_goal/src/nav_goal/movebase_client.py

```diff
@@ -0,0 +1,38 @@
+import rospy
+import actionlib
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from tf.transformations import quaternion_from_euler
+
+def movebase_client(x, y, yaw):
+  # Create an action client called "move_base" with action definition file "MoveBaseAction"
+  client = actionlib.SimpleActionClient('move_base', MoveBaseAction)
+  # Waits until the action server has started up and started listening for goals.
+  client.wait_for_server()
+
+  # Creates a new goal with the MoveBaseGoal constructor
+  goal = MoveBaseGoal()
+
+  goal.target_pose.header.frame_id = "base_link"
+  goal.target_pose.header.stamp = rospy.Time.now()
+
+  # Set goal position
+  goal.target_pose.pose.position.x = x
+  goal.target_pose.pose.position.y = y
+
+  # Convert yaw to quaternion and set pose
+  q = quaternion_from_euler(0, 0, yaw)
+  goal.target_pose.pose.orientation.x = q[0]
+  goal.target_pose.pose.orientation.y = q[1]
+  goal.target_pose.pose.orientation.z = q[2]
+  goal.target_pose.pose.orientation.w = q[3]
+
+  # Sends goal and waits until the action is completed (or aborted if it is impossible)
+  client.send_goal(goal)
```

Comment:
I think it's important to provide feedback to the user telling him/her if the goal has been sent (and what values where used)

Reply:
The user will get the response from the client, and if no response then it will get the error message. If the goal is reached it will say that, if it can't be it will say that. What other message do you want to add?

Reply:
Like once the action server receives the nav goal message. I just want feedback knowing that what I sent was actually sent

Reply:
Sounds good, I will add a 'sent' message

Reply:
Done